packages/components/psammead-brand/CHANGELOG.md

```diff
@@ -3,6 +3,7 @@
 <!-- prettier-ignore -->
 | Version | Description |
 | ------- | ----------- |
+| 1.2.0   | [PR#480](https://github.com/bbc/psammead/pull/480) Allow the SVG to dynamically scale between the minimum and maximum widths passed in as props, also bump psammead-assets@0.1.9. Also reduce banner height and left-right padding under 600px. |
```

Comment:
You're adding new required props & making an existing optional prop required.  Currently this PR has this as a minor version increase, so the changes will automatically be pulled into Simorgh on the next build and we'll see the Brand breaking since there's no `minWidth`, `maxWidth` etc defined in the consuming Simorgh application.  For this reason, this PR should have a major version increase to `2.0.0`.

Reply:
Thanks, slipped my mind